### ISSUE TITLE
protontricks doesnt recognize proton installation.

### ISSUE THREAD

**Issue by bugpoop** [Matoking/protontricks#74]
Hello, I have Proton 3.7 in my steam folders, and I have ran a steam play requiring game before on the computer, there is no way that proton is not installed however I still get the error(s) below:
 `protontricks (ERROR): No Proton installation found in config.vdf
protontricks (ERROR): Active Proton installation could not be found automatically.
Proton installation could not be found!`

Some help would be greatly appreciated.

**Comment by Matoking** [comment]
Could you tell me the following details about your environment:

 - Distro: [e.g. Ubuntu 20.04, Arch Linux, ...]
 - Protontricks version: run `protontricks --version` to print the version
 - Steam version: check if you're running Steam beta; this can be checked in _Steam_ -> _Settings_ -> _Account_ -> _Beta participation_

Also, running the command with the `-v` flag could give more useful information about what is going wrong. For example, if you're trying to run `protontricks 123456 xact`, run `protontricks -v 123456 xact` instead.

**Comment by bugpoop** [comment]
Im on ubuntu 20.04 with ptricks 1.4.2 and I am not running steam beta

UPDATE: I tried installing another game and forced steam to install the up-to-date proton 5.1.3 and now I am getting a different error.

`Traceback (most recent call last):
  File "/home/emre/.local/bin/protontricks", line 11, in <module>
    load_entry_point('protontricks==1.4.2', 'console_scripts', 'protontricks')()
  File "/home/emre/.local/pipx/venvs/protontricks/lib/python3.8/site-packages/protontricks/cli.py", line 193, in main
    run_command(
  File "/home/emre/.local/pipx/venvs/protontricks/lib/python3.8/site-packages/protontricks/util.py", line 164, in run_command
    wine_bin_dir = create_wine_bin_dir(proton_app=proton_app)
  File "/home/emre/.local/pipx/venvs/protontricks/lib/python3.8/site-packages/protontricks/util.py", line 56, in create_wine_bin_dir
    binaries = list((
  File "/usr/lib/python3.8/pathlib.py", line 1118, in iterdir
    for name in self._accessor.listdir(self):
FileNotFoundError: [Errno 2] No such file or directory: '/home/emre/.steam/steam/steamapps/common/Proton 5.13/dist/bin'
`

**Comment by bugpoop** [comment]
UPDATE: fixed the issue mentioned above by unpacking the proton_dist.tar and creating a directory titled "dist"

**Comment by Matoking** [comment]
Proton should extract the required files once you have started a game at least once. You shouldn't need to extract `proton_dist.tar` yourself.

---

Could you check what settings are enabled in the Steam Play Settings (you can find them in Steam under *Settings* -> *Steam Play*)? Is "Enable Steam Play for all other titles" checked and does the "Run other titles with" dropdown menu have a Proton version selected?

**Comment by bugpoop** [comment]
it was not checked. it installed proton 3.7 for battlefront 2 2005, and then I checked it and made it the latest version to force installing proton 5.1.3

**Comment by Matoking** [comment]
OK, I checked and Protontricks won't be able to find anything in `config.vdf` if Steam Play is only enabled for supported titles. This is rarely a problem since there's usually no need to use Protontricks when only playing supported titles, but I've clarified the error message to make sure the option is enabled.

Does the original issue still happen once you have checked the "Enable Steam Play for all other titles" and tried running a Protontricks command *after* you've launched the game once?
